src/resources/charge_ext.rs

```diff
@@ -1,12 +1,67 @@
 use crate::config::{Client, Response};
-use crate::ids::ChargeId;
+use crate::ids::{BankAccountId, CardId, ChargeId, SourceId, TokenId};
 use crate::params::Object;
 use crate::resources::{Charge, Rule};
 use serde_derive::{Deserialize, Serialize};
 
+/// The set of PaymentSource parameters that can be used to create a charge.
+/// 
+/// For more details see [https://stripe.com/docs/api/charges/create#create_charge-source](https://stripe.com/docs/api/charges/create#create_charge-source).
+#[derive(Clone, Debug)]
+pub enum PaymentChargeParams {
```

Comment:
Since this is for the `Charge` struct's `source`   field, would it make sense to call it `ChargeSourceParams`?

Reply:
Yes, updated!